### ISSUE TITLE
1.17 dupe hanging_roots with rooted_dirt

### ISSUE THREAD

**Issue by tommasobenatti** [GriefPrevention/GriefPrevention#1535]
### What happened:
With an hoe enchanted with silk touch inside a zone claimed that you haven't the trust if you try to remove the hanging roots from the rooted dirt it gives you illimitates hanging roots

### What was expected:
To not give you the hanging_roots

### Steps to reproduce:
1. Stay inside a claim that you haven't the access
2. Grab an hoe enchanted with silk touch
3. try to right-click a rooted dirt
4. You have illimitate hanging_roots
video of the bug (https://www.youtube.com/watch?v=gJo6gsv5FIs)

### Server and GriefPrevention version:
Paste of /version: 
This server is running Airplane version git-Airplane-"ec7d3d0" (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT) 

Paste of /ver griefprevention:
GriefPrevention version ${git.commit.id.describe}
(Latest commit e8c5b37)

### Stack trace/error or server log

```
No error occurred in console
```

### GriefPrevention config.yml

```yml

https://drive.google.com/file/d/15qWdDsfRWafjyIMyeDDHVlbeRkrjxJUG/view?usp=drivesdk

```

### Plugin list (if applicable):

Plugins (49): AdvancedDevelopmentLibrary, AngelChest, BetterJails, BetterRTP, BossShopPro, BungeeGuard, BuycraftX, ChatReaction, ChatSentinel, ChestShop, CoreProtect, CrateReloaded, DeluxeChat, Elevator, EntityDetection, Essentials, EssentialsSpawn, ExperienceCore, EZColors, FarmLimiter, GriefPrevention, HolographicDisplays, Images, LuckPerms, MeteoritesPro, MineableSpawners, Minepacks, MoneyFromMobs, NametagEdit, NerfPhantoms, OpenInv, PL-Hide-Pro, PlaceholderAPI, PlugMan, ProtocolLib, Rankup, ReviveMe, SkinsRestorer, SuperbVote, SuperVanish, TitleManager, TradeSystem, TreeAssist, UltimateCatcher, Vault, Votifier, Voting, WorldEdit, WorldGuard

### Debugging steps attempted

- [ ] (If applicable) I attempted running the server without GriefPrevention installed
  - [ ] (If applicable) The problem does _not_ occur if GriefPrevention is removed from the server.
- [x] I attempted running only GriefPrevention on the server
  - [x] Issue still occurs even if GriefPrevention is the only plugin running
- [x] I attempted testing for the issue on a new server
  - [x] Issue still occurs even after testing on a new server
- [x] I attempted testing for the issue with the vanilla client (no mods)
  - [x] Issue still occurs even after testing with the vanilla Minecraft client

**Comment by brasilcraft54** [comment]
![2021-08-26_19 32 07](https://user-images.githubusercontent.com/67756407/131044844-51b94718-c168-48eb-86cf-447a5642dd79.png)

I have the same problem. 

paper 1.17.1

version: '1347291'
api-version: '1.17'

**Comment by bigpresh** [comment]
Sorry you've not had a response yet, and thanks for the helpful, detailed report, even with a video!

I believe this will be fixed by PR #1541
